test/benchmark/bench_linux_and_aster.sh

```diff
@@ -113,6 +119,7 @@ if [ -z "$2" ] || [ "$2" = "null" ]; then
 else
     BENCHMARK_TYPE="$2"
 fi
+ASTER_SCHEME="$3"
```

Comment:
`BENCHMARK_TYPE` is defaulted to `guest_only`. In the workflow, `jq` will extract null as the result, but when we test it locally, e.g., `bash test/benchmark/bench_linux_and_aster.sh lmbench/ext2_copy_files_bw`, it is empty, so the `ASTER_SCHEME `also needs to make the same judgment.

Reply:
Revised. And I put the judgement to L63 since the command needs further processing (add a prefix).